go/vt/vtgate/gateway.go

```diff
@@ -128,21 +128,55 @@ type GatewayEndPointCacheStatus struct {
 	AvgLatency float64 // in milliseconds
 }
 
-// NewGatewayEndPointStatusAggregator creates a GatewayEndPointStatusAggregator.
-func NewGatewayEndPointStatusAggregator() *GatewayEndPointStatusAggregator {
-	gepsa := &GatewayEndPointStatusAggregator{
-		qiChan: make(chan *queryInfo, 10000),
+var (
+	// aggrChan buffers queryInfo objects to be processed.
+	aggrChan chan *queryInfo
+	// muAggr protects below vars.
+	muAggr sync.Mutex
+	// aggregators holds all Aggregators created.
+	aggregators []*GatewayEndPointStatusAggregator
+)
+
+func init() {
+	// init global goroutines to aggregate stats.
+	aggrChan = make(chan *queryInfo, 100000)
```

Comment:
let's make the 100k and the following 10 below constants at the beginning of the file.